egs/visualization/SingVisio/README.md

```diff
@@ -57,12 +59,32 @@ The user inference of **SingVisio** is comprised of five views:
 
 ## Detailed System Introduction of SingVisio
 
-For a detailed introduction to **SingVisio** and user instructions, please refer to [this online document](https://x8gvg3n7v3.feishu.cn/docx/IMhUdqIFVo0ZjaxlBf6cpjTEnvf?from=from_copylink) (with animation) or [offline document](../../../visualization/SingVisio/System_Introduction_of_SingVisio.pdf) (without animation).
+For a detailed introduction to **SingVisio** and user instructions, please refer to [this document](../../../visualization/SingVisio/System_Introduction_of_SingVisio_V2.pdf).
```

Comment:
It would be better to remove the demo link in the PDF version of the system introduction because the link is inaccessible.

Reply:
The accessible link is already attached in the PDF above the video widget and is titled "Online Link." (Tested working without login requirement)

<img width="502" alt="image" src="https://github.com/user-attachments/assets/de79b593-0797-4958-b7e9-2fb53420224b">